server/SwipeDecision.py

```diff
@@ -0,0 +1,80 @@
+import mysql.connector
+from mysql.connector import errorcode
+from connect import connectToDB
+
+
+def getPotentialMatchList(currentUserId):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            PotentialMatchQuery = "SELECT shownUser FROM PotentialMatch WHERE currentUser = %s and matchDecision IS NULL"
+            userID = (currentUserId,)    
+            cursor.execute(PotentialMatchQuery, userID)
+            potentialListId = [i[0] for i in cursor.fetchall()]
+
+            connection.commit()
+            cursor.close()
+            return {"response": "Success", 
+            "potentialListId": potentialListId}
+    
+    except mysql.connector.Error as err:
+        return {"response": err.msg }
+
+    return{"response": "Something went wrong"}
+
+def showProfile(shownUserId):
+    try:
+        connection = connectToDB()
+        if(connection != False):
+            cursor = connection.cursor(buffered=True)
+            shownUserProfileQuery = "SELECT firstname, interests, description, age, gender, workPlace FROM Profile WHERE userId = %s"
+            userID = (shownUserId,) 
+            cursor.execute(shownUserProfileQuery, userID)
+            if(cursor.rowcount<1):
+                return {"response": "Error loading user profile"}
+            result = cursor.fetchall()
+
+            for row in result: 
+                firstName = row[0]
+                interests = row[1]
+                description = row[2]
```

Comment:
If this is just for 1 user since the userId is unique, why do we still have to iterate through the result?